hbase-it/src/test/java/org/apache/hadoop/hbase/mapreduce/IntegrationTestBulkLoad.java

```diff
@@ -373,7 +375,11 @@ public void close() throws IOException {
       taskId = taskId + iteration * numMapTasks;
       numMapTasks = numMapTasks * numIterations;
 
-      long chainId = Math.abs(new Random().nextLong());
+      // ensure chainId is positive
+      long chainId = RNG.nextLong();
+      if (chainId < 0) {
```

Comment:
Ditto, not the correct way for fixing the abs problem...

Reply:
As explained by the help text for the spotbugs finding the issue is Math.abs(INT_MIN) can return INT_MIN. So if ABS is supposed to ensure a positive number, it will not always do so. The fix is to change the sign of the value if it is negative and should not be negative.